python-avd/pyavd/_cv/client/models.py

```diff
@@ -0,0 +1,87 @@
+# Copyright (c) 2025 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+from typing import Literal, NamedTuple, Self
+
+from pyavd._cv.api.arista.tag.v2 import ElementType, Tag, TagAssignment
+
+ELEMENT_TYPE_MAP = {
+    "device": ElementType.DEVICE,
+    "interface": ElementType.INTERFACE,
+    "unspecified": ElementType.UNSPECIFIED,
+}
+
+REVERSED_ELEMENT_TYPE_MAP = {
+    ElementType.DEVICE: "device",
+    ElementType.INTERFACE: "interface",
+    ElementType.UNSPECIFIED: "unspecified",
+}
+
+
+class TagTuple(NamedTuple):
```

Comment:
```suggestion
class Tag(NamedTuple):
```

or CvTag or something.